### ISSUE TITLE
Question Regarding Table 6 in the paper

### ISSUE THREAD

**Issue by jzhang38** [thunlp/PL-Marker#2]
Hi,

Thanks for your excellent work!

I have 2 questions regarding table 6 in the paper:

1. In table 6 you report the score of 69.1 for PL-Marker. Did you obtain this number under the single-sentence setting or cross-sentence setting? As the scores of PURE that you report in the same table are obtained under the single-sentence setting.

2.  "All models adopt the same entity input from the entity model of PURE". The number 69.1 in table 6 is higher than 68.8 which is reported in table 4 using BERT-base. Can I say the entity model of PURE performs better than your proposed entity model, at least under this specific setting?

**Comment by YeDeming** [comment]
1. Sorry for the mistake. I report the score of PL-Marker under cross-sentence setting. But I use the result of PURE under single-sentence setting in Table 6. I will correct the results as soon as possible.

2. Yes. As shown in Table 4. With bert-base-uncased, the entity model of PURE gets 90.1 F1, which is better than our model with 89.7 F1. The entity model of PURE uses span boundary and the span width to classify. I have reproduced PURE's entity score with their released code with seed 0~4. But I failed to get similar result with my implementation and seed 42~47. Moreover, with albert-xxlarge, both my implementing span entity model and PL-Marker entity model outperform that of PURE.
